bitnami/external-dns/values.yaml

```diff
@@ -1205,11 +1205,17 @@ extraVolumes: []
 ## @param extraVolumeMounts A list of volume mounts to be added to the pod
 ##
 extraVolumeMounts: []
-## @param podDisruptionBudget Configure PodDisruptionBudget
-## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
-##
-
-podDisruptionBudget: {}
+## Pod Disruption Budget configuration
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
+## DEPRECATED: podDisruptionBudget will be removed in a future release - Please use pdb instead
```

Comment:
You shouldn't remove if you're deprecating it: 
```suggestion
## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
## DEPRECATED: podDisruptionBudget will be removed in a future release - Please use pdb instead
podDisruptionBudget: {}
```

Reply:
Suggestion applied. I also set `pdb.create=true` by default